packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -141,11 +151,17 @@ class CodeSizeController {
   void loadDiffTreeFromJsonFiles(
     DevToolsJsonFile oldFile,
     DevToolsJsonFile newFile,
+    void Function(String error) onError,
   ) {
-    if (oldFile == null || newFile == null) return;
+    if (oldFile == null || newFile == null) {
+      return;
+    }
 
-    changeOldDiffSnapshotFile(oldFile);
-    changeNewDiffSnapshotFile(newFile);
+    if (oldFile.isApkFile != newFile.isApkFile) {
```

Comment:
what about v8 snapshot vs instruction sizes? we should probably add a check for that too

Reply:
Done